deeplake/core/dataset/dataset.py

```diff
@@ -3185,18 +3182,15 @@ def load_view(
         Raises:
             KeyError: if view with given id does not exist.
         """
+        view = self.get_view(id)
```

Comment:
@farizrahman4u I tested the code and it works, but why is this function not running infinitely, since it's running recursively with no offramp?

Reply:
load_view and get_view are different methods. There is no recursion.